### ISSUE TITLE
Language server failing in 11.6

### ISSUE THREAD

**Issue by adsbhatial** [vscode-abl/vscode-abl#253]
Hi team,

I'm trying to use the extension in 11.6. But the language server is failing to start. 

```
[2024-11-27T11:11:24,275] [ERROR] [application] ### OE Client #0 ended with exit code 2 ###
[2024-11-27T11:11:24,276] [ERROR] [application] ### OE Client #0 standard output can be found in <WORKSPACE>\.builder\stdout0.log
[2024-11-27T11:11:24,276] [ERROR] [application] ### OE Client #0 log file can be found in <WORKSPACE>\.builder\clientlog0.log
[2024-11-27T11:11:24,602] [ERROR] [application] ### OE Client #0 ended with exit code 2 ###
[2024-11-27T11:11:24,602] [ERROR] [application] ### OE Client #0 standard output can be found in <WORKSPACE>\.builder\stdout0.log
[2024-11-27T11:11:24,602] [ERROR] [application] ### OE Client #0 log file can be found in <WORKSPACE>\.builder\clientlog0.log
[2024-11-27T11:11:25,272] [INFO] [application] Read typeInfoRoot cache in 0 ms
[2024-11-27T11:11:25,283] [INFO] [application] Read propathTypeInfoRoot cache in 2 ms
[2024-11-27T11:11:25,283] [INFO] [application] Read catalog cache in 0 ms
[2024-11-27T11:11:25,384] [INFO] [application] TypeInfo cache cleanup in 101 ms 
[2024-11-27T11:11:25,763] [INFO] Not restarting session #0 as it crashed 3 times in the last 10 seconds
[2024-11-27T11:11:25,764] [ERROR] [application] ### OE Client #0 ended with exit code 2 ###
[2024-11-27T11:11:25,764] [ERROR] [application] ### OE Client #0 standard output can be found in <WORKSPACE>\.builder\stdout0.log
[2024-11-27T11:11:25,764] [ERROR] [application] ### OE Client #0 log file can be found in <WORKSPACE>\.builder\clientlog0.log
```

Checked the builder clientlog0.log and found

```
[24/11/27@11:11:25.720-0500] P-026180 T-010780 1 4GL -- Unknown attribute exit-code used in widget:attribute phrase. (3406)
[24/11/27@11:11:25.720-0500] P-026180 T-010780 1 4GL -- ** <WORKSPACE>\.builder\ls\thread.p Could not understand line 162. (196)
[24/11/27@11:11:25.728-0500] P-026180 T-010780 1 4GL -- ** Unable to run startup procedure <WORKSPACE>\.builder\ls\thread.p. (492)
```

Checked stdout0.log and found
```
Unknown attribute exit-code used in widget:attribute phrase. (3406)
** <WORKSPACE>\.builder\ls\thread.p Could not understand line 104. (196)
Unknown attribute exit-code used in widget:attribute phrase. (3406)
** <WORKSPACE>\.builder\ls\thread.p Could not understand line 162. (196)
** Unable to run startup procedure <WORKSPACE>\.builder\ls\thread.p. (492)
```

Checked the thread.p and found SESSION:EXIT-CODE being used. 
EXIT-CODE was introduced in 11.7.3. So it's failing while being run in 11.6

https://docs.progress.com/bundle/openedge-new-info-1173/page/New-SESSION-EXIT-CODE.html.

I think the thread.p is being generated by the Language server. 

Could there be a fix that checks to add EXIT-CODE only if the OpenEdge version is 11.7 and above?

Thanks.

**Comment by gquerret** [comment]
That was already reported last week in our internal bugtracker, but the commit has not been pushed yet. I'll try to push it in the prerelease stream this week. Note that 11.6 is untested / unsupported, there may be other issues with this version.

**Comment by gquerret** [comment]
Fixed in 1.15.5

**Comment by adsbhatial** [comment]
Thanks Gilles
